apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/editor/_components/Editor/Simple/NumberEvaluationEditor/index.tsx

```diff
@@ -0,0 +1,208 @@
+        ) as string,
+      },
+      configuration: {
+        type: EvaluationResultableType.Number,
+        minValue: Number(formData.get('minValue')),
+        maxValue: Number(formData.get('maxValue')),
+        minValueDescription: formData.get('minValueDescription') as string,
+        maxValueDescription: formData.get('maxValueDescription') as string,
+      },
+    })
+    if (error) {
+      toast({
+        title: 'Error',
+        description: error.message,
+        variant: 'destructive',
+      })
+    }
+  }
+
+  return (
+    <div className='flex flex-col gap-y-2 h-full'>
+      <div className='flex flex-row items-center justify-between'>
+        <Text.H4M>{evaluation.name}</Text.H4M>
+        <Button fancy form='simple-number-evaluation-editor' type='submit'>
+          Save changes
+        </Button>
+      </div>
+      <form
+        className='bg-backgroundCode flex flex-grow flex-col gap-y-4 p-4 rounded-lg border'
+        id='simple-number-evaluation-editor'
+        onSubmit={onSubmit}
+      >
+        <FormField>
+          <ProviderModelSelector
+            modelDisabled={!selectedProvider}
+            modelOptions={modelOptions}
+            onModelChange={onModelChange}
+            onProviderChange={onProviderChange}
+            providerDisabled={!providerOptions.length}
+            providerOptions={providerOptions}
+            selectedModel={selectedModel}
+            selectedProvider={selectedProvider}
+          />
+        </FormField>
+        <FormField label='Evaluation objective'>
+          <Input
+            name='objective'
+            defaultValue={metadata.objective}
+            placeholder='The main objective of the evaluation'
+          />
+        </FormField>
+        <FormFieldGroup>
+          <FormField label='Min Value'>
+            <Input
+              type='number'
+              name='minValue'
+              defaultValue={resultConfiguration.minValue}
+            />
+          </FormField>
+          <FormField label='Min value description'>
+            <Input
+              name='minValueDescription'
+              defaultValue={resultConfiguration.minValueDescription ?? ''}
+              placeholder='Description of the min value'
+            />
+          </FormField>
+        </FormFieldGroup>
+        <FormFieldGroup>
+          <FormField label='Max Value'>
+            <Input
+              type='number'
+              name='maxValue'
+              defaultValue={resultConfiguration.maxValue}
+            />
+          </FormField>
+          <FormField label='Max value description'>
+            <Input
+              name='maxValueDescription'
+              defaultValue={resultConfiguration.maxValueDescription ?? ''}
+              placeholder='Description of the max value'
+            />
+          </FormField>
+        </FormFieldGroup>
+        <div className='flex flex-col gap-2 flex-grow'>
+          <Label>Additional instructions</Label>
+          <textarea
```

Comment:
We have a nice auto grow textarea atom
https://github.com/latitude-dev/latitude-llm/blob/main/packages/web-ui/src/ds/atoms/TextArea/index.tsx

Reply:
yeah i tried that component wasn't working out for me, it just wasn't taking the full height so I'm not sure what it's supposed to do?

Reply:
Fix it no?

Reply:
Also I noticed you do `<FormField><Input /></FormField>`. No need just pass `label` to `<Input />`. It has the form field included

Reply:
Let's comment over here
https://github.com/latitude-dev/latitude-llm/pull/578